core_backend/app/question_answer/speech_components/external_voice_components.py

```diff
@@ -57,16 +52,12 @@ async def transcribe_audio(audio_filename: str) -> str:
 
 async def generate_tts_on_gcs(
```

Comment:
You are not saving the output too GCS right? Then you should probably change the name

Reply:
oh yes! before we were saving it to gcs in that function itself but now i changed it to make it more modularised, i will change it again